packages/airnode-examples/integrations/authenticated-coinmarketcap/config.example.json

```diff
@@ -36,7 +36,7 @@
     },
     "logFormat": "plain",
     "logLevel": "INFO",
-    "nodeVersion": "0.5.0",
+    "nodeVersion": "0.4.0",
```

Comment:
Basically, the plan is to update these before releasing a new version (will be added as a step in the versioning flow). Gladly, this will only need to be updated in examplatory `config.json` files (in deployer, node and examples).